x/distribution/keeper/allocation.go

```diff
@@ -13,7 +13,7 @@ import (
 // AllocateTokens handles distribution of the collected fees
 func (k Keeper) AllocateTokens(
 	ctx sdk.Context, sumPreviousPrecommitPower, totalPreviousPower int64,
-	previousProposer sdk.ConsAddress, previousVotes []abci.VoteInfo,
+	previousProposer sdk.ConsAddress, bondedValidatorsVoteInfo []abci.VoteInfo,
```

Comment:
```suggestion
	previousProposer sdk.ConsAddress, bondedVotes []abci.VoteInfo,
```

Reply:
in go we don't prefer concise names and we don't need to repeat a type name in the variable name.